declarations/Buffer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Buffer",
+  "terms": {
+    "Terms of Service": {
+      "fetch": "https://buffer.com/legal",
+      "select": [
+        "[aria-labelledby=current-terms-of-use]"
+      ],
+      "remove": [
+        "[href=\"/legal/previous\"]"
+      ]
+    },
+    "Privacy Policy": {
+      "fetch": "https://buffer.com/legal",
+      "select": [
+        "[aria-labelledby=privacy-policy]",
+        "[aria-labelledby=crpn]",
```

Comment:
We are not concerned by California Resident related notice

Reply:
True, but isn't it part of the privacy policy anyways?